middle-pgsql.cpp

```diff
@@ -245,20 +156,34 @@ pg_result_t middle_query_pgsql_t::exec_prepared(char const *stmt,
     return exec_prepared(stmt, buffer);
 }
 
+pg_result_t middle_pgsql_t::exec_prepared(char const *stmt, osmid_t osm_id) const
+{
+    char buffer[64];
+    sprintf(buffer, "%" PRIdOSMID, osm_id);
```

Comment:
Opened #916 to cover the existing usage of `sprintf`

Reply:
Replaced all 4 occurrences I could find with snprintf.